docs/recipes/typescript.md

```diff
@@ -2,7 +2,7 @@
 
 ### Examples
 
-- [Counter](../../examples/ts/count)
+- [Counter](github.com/rematch/rematch/tree/master/examples/ts/count/)
```

Comment:
I'd suggest adding a protocol to this, as otherwise it could be interpreted as linking to `<current-url>/github.com/rematch/rematch/tree/master/examples/ts/count/`.